letta/schemas/memory.py

```diff
@@ -213,6 +213,7 @@ def __init__(self, persona: str, human: str, limit: int = CORE_MEMORY_BLOCK_CHAR
             human (str): The starter value for the human block.
             limit (int): The character limit for each block.
         """
+        # TODO: Should these be CreateBlocks?
```

Comment:
I think we need to get rid of this concept... this was intended to be a nice way for the client to initialized a specific memory "type", but over time it seems like the only customization we need is at the block level, and memory is just a list of block objects and memory tools. We should maybe get rid of these `BasicBlockMemory` extensions, and definitely get rid of the free floating `Block` since that's dangerous under the new model.

Reply:
will do in a follow up PR